resources/js/components/RegisterForm.tsx

```diff
@@ -33,10 +36,31 @@ export default function RegisterForm({ onRegister }) {
     })
 
     async function register(values: z.infer<typeof formSchema>) {
-        const { data, errors } = await post("/register", values)
+        setIsSubmitting(true);
+        setError(null);
```

Comment:
we never use `error` anywhere else, I think we can remove `error` and `setError` for now

Reply:
I removed setError and removed all the throw and catch conditions